ports/stm32/mboot/mboot_pack_dfu.py

```diff
@@ -135,6 +140,8 @@ def encrypt(keys, data):
 
 
 def sign(keys, data):
+    if not hasattr(keys, "sign_sk"):
+        raise RuntimeError("Packing a dfu requires a secret key")
```

Comment:
`RuntimeError` is usually reserved for internal interpreter errors, like stack overflow.  Instead please use `Exception` with lower-case for the first word, to match other exceptions in this file.

Reply:
Fixed